Source/AwsCommonRuntimeKit/sdkutils/endpoint/EndpointProperty.swift

```diff
@@ -84,17 +84,17 @@ extension EndpointProperty: Decodable {
     }
 }
 
-extension Dictionary where Key == String, Value == EndpointProperty {
-    /// Converts EndpointProperty to a dictionary of `String`: `AnyHashable`
-    /// - Returns: Dictionary of `String`: `AnyHashable`
-    func toStringHashableDictionary() -> [String: AnyHashable] {
-        var dict: [String: AnyHashable] = [:]
-        for (key, value) in self {
-            dict[key] = value.toAnyHashable()
-        }
-        return dict
-    }
-}
+//extension Dictionary where Key == String, Value == EndpointProperty {
+//    /// Converts EndpointProperty to a dictionary of `String`: `AnyHashable`
+//    /// - Returns: Dictionary of `String`: `AnyHashable`
+//    func toStringHashableDictionary() -> [String: AnyHashable] {
```

Comment:
The function seems not used. Is it safe to remove it?

Reply:
Thanks, removed.